pkg/controller/cassandracluster/testdata/cassandracluster-2DC-ManualSeedlist.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: "db.orange.com/v1alpha1"
+kind: "CassandraCluster"
+metadata:
+  name: cassandra-demo
+  labels:
+    cluster: k8s.pic
+  namespace: ns
+spec:
+  nodesPerRacks: 1
+  cassandraImage: cassandra:latest
+  imagePullSecret:
+    name: advisedev # To authenticate on docker registry
+  rollingPartition: 0
+  service:
+    annotations:
+      external-dns.alpha.kubernetes.io/hostname: my.custom.domain.com.
+  pod:
+    annotations:
+      exemple.com/test: my.custom.annotation
+    tolerations:
+      - key: my_custom_taint
+        operator: Exists
+        effect: NoSchedule
+  dataCapacity: "3Gi"
+  dataStorageClass: "local-storage"
+  hardAntiAffinity: false
+  deletePVC: true
+  autoPilot: true
+  resources:         
+    requests:
+      cpu: '1'
+      memory: 2Gi
+    limits:
+      cpu: '1'
+      memory: 2Gi
+  topology:
+    dc:
+      - name: dc1
+        labels:
+          location.dfy.orange.com/site : mts
+        rack:
+          - name: rack1
+            labels: 
+              location.dfy.orange.com/street : street1
+          - name: rack2
+            labels: 
+              location.dfy.orange.com/street : street2
+      - name: dc2
+        nodesPerRacks: 1
+        labels: 
+          location.dfy.orange.com/site : mts
+        rack:
+          - name: rack1
+            labels: 
+              location.dfy.orange.com/street : street3
+status:
+  seedlist:
```

Comment:
Like I said earlier I think we can get rid of that sample deployment file. However I have a question. Why is the seedlist in the status key ? It's surprising ...

Reply:
Yes it was the subjezct of the issue 144 which was closed by JAL without answering the initial question.
We discussed it with Seb and thought that to put it in Spec would be too much of a giveway for non casskop control of the seedlist. So we left it in the status key

Reply:
gotcha